src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAYSIZE = 6;
```

Comment:
Consider giving a more descriptive name to the constant 'ARRAYSIZE'. For example, 'FIGURES_ARRAY_SIZE' would better describe what the constant represents and how it's being used.